docs/dev/guides/backend-routes.md

```diff
@@ -26,27 +26,39 @@ and is configured through annotations.
 // src/Controller/BackendController.php
 namespace App\Controller;
```

Comment:
```suggestion
namespace App\Action;
```

Reply:
👎  as far as I know, these are always `Controller` in Symfony

Reply:
» https://symfony.com/doc/current/controller/service.html#invokable-controllers

Reply:
Hm, I was sure that Symfony used `App\Action` there, but evidently they don't. Not sure where I got the recommendation from then.

There was a PR about it, but it did not get merged. https://github.com/symfony/symfony-docs/pull/8153/files

According to the discussion, Symfony is net yet promoting the ADR pattern within the new `Action` namespace.

Reply:
» https://github.com/contao/docs/pull/227